integrations/upgrade/js-config.test.ts

```diff
@@ -108,6 +159,54 @@ test(
         --font-size-base: 1rem;
         --font-size-base--line-height: 2rem;
 
+        --width-*: initial;
+        --width-0: 0%;
+        --width-1: 0.25rem;
+        --width-2: 0.5rem;
+        --width-3: 0.75rem;
+        --width-4: 1rem;
+        --width-5: 1.25rem;
+        --width-6: 1.5rem;
+        --width-8: 2rem;
+        --width-10: 2.5rem;
+        --width-11: 2.75rem;
+        --width-12: 3rem;
+        --width-16: 4rem;
+        --width-24: 6rem;
+        --width-32: 8rem;
+        --width-40: 10rem;
+        --width-48: 12rem;
+        --width-64: 16rem;
+        --width-80: 20rem;
+        --width-96: 24rem;
+        --width-128: 32rem;
+        --width-px: 1px;
+        --width-auto: auto;
+        --width-1_5: 0.375rem;
+        --width-2_5: 0.625rem;
+        --width-3_5: 0.875rem;
+        --width-full: 100%;
+        --width-1/2: 50%;
```

Comment:
I’m pretty sure the slashes will need to be escaped, they aren’t valid characters in CSS variable names.

Can we add a higher level test that includes theme values like this and test the CSS output when using a utility that is derived from a key with a slash?

Reply:
Seems like this doesn't at all right now including for things like:

```css
@import "tailwindcss";

@theme {
  --color-red-1\/2: red;
}
```

Looking into fixing this ASAP.

Reply:
Hmm okay this is going to get tricky, we parse this as a modifier right now 🤔

Reply:
Trying to get this to work:

```ts
let input = css`
  @tailwind utilities;
  @plugin "my-plugin";

  @theme {
    --width-1: 0.25rem;
    --width-1\.5: 0.375rem;
    --width-1\/2: 50%;
    --width-2_5: 0.625rem;
  }
`

let compiler = await compile(input, {
  loadModule: async (id, base) => {
    return {
      base,
      module: plugin(function ({ matchUtilities, theme }) {
        console.dir(theme('width'))
        matchUtilities(
          { 'my-width': (value) => ({ 'width:': value }) },
          { values: theme('width') },
        )
      }),
    }
  },
})
```